middleware/verify-token.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require("jsonwebtoken")
+
+function verifyToken(req, res, next) {
+	try {
+		const token = req.headers.authorization.split(" ")[1]
```

Comment:
I'm unsure if this code is doing what we need it to do. To me this looks more like the isLoggedIn function than the isAuthorized. I think we should speak to Brian to try get a better understanding of how this JWT stuff actually functions. Recommend you move on to a different card for now and leave this in blocked.